src/azure-cli/azure/cli/command_modules/appservice/commands.py

```diff
@@ -498,8 +498,19 @@ def load_command_table(self, _):
         g.custom_command('stop', 'stop_webapp')
         g.custom_command('start', 'start_webapp')
         g.custom_command('restart', 'restart_webapp')
+        g.generic_update_command('update', getter_name="get_functionapp", setter_name='set_functionapp', exception_handler=update_function_ex_handler_factory(),
+                                 custom_func_name='update_functionapp', getter_type=appservice_custom, setter_type=appservice_custom, command_type=webapp_sdk)
 
     with self.command_group('logicapp', custom_command_type=logicapp_custom) as g:
         g.custom_command('create', 'create_logicapp', exception_handler=ex_handler_factory())
         g.custom_command('list', 'list_logicapp', table_transformer=transform_web_list_output)
         g.custom_show_command('show', 'show_logicapp', table_transformer=transform_web_output)
+        g.custom_command('scale', 'scale_logicapp', exception_handler=ex_handler_factory())
+
+    with self.command_group('logicapp config appsettings', custom_command_type=logicapp_custom) as g:
+        g.custom_command('list', 'get_logicapp_app_settings', exception_handler=empty_on_404)
```

Comment:
for all LA commands is there a reason we need to have explicit methods - is there a reason we cannot just point these to function apps?  or atleast in a way we can re-use as much of the code as possible?

Reply:
We currently import and reuse the functionapp commands in the logicap custom file. I did it this way so that we can easily update the logicapp commands in the future if they diverge from their respective functionapp definitions.

Reply:
Logic app standard is built on Functions, so they won't diverge. What was done initially is not right, so we shouldn't maintain copies of the code, so please use functionapp methods here.

Reply:
Thanks for explaining this. This is good.